### DISCUSSION TITLE
Mark as UNwatched in Trakt?

### DISCUSSION THREAD

**Discussion by michaeldrouin** [Taxel/PlexTraktSync#1531] [Q&A] [answered]
Is it possible to syn UNwatched status from Plex to Trakt? i had some stuff (about 100 movies) erroneously marked as "Watched" in Plex and ran the sync. I went back in Plex and marked them to "Unwatched". is there a way to have it remove the "watched" status in Trakt is Plex says "Unwatched"?

**Comment by simonc56** [comment, marked as answer]
There is no `unwatched_at` event in Plex history.
Only `watched_at` events.

See #1080
